ext/pdo_sqlite/config.w32

```diff
@@ -3,23 +3,15 @@
 ARG_WITH("pdo-sqlite", "for pdo_sqlite support", "no");
 
 if (PHP_PDO_SQLITE != "no") {
-	EXTENSION("pdo_sqlite", "pdo_sqlite.c sqlite_driver.c sqlite_statement.c", null, "/DSQLITE_THREADSAFE=" + (PHP_ZTS == "yes" ? "1" : "0") + " /D SQLITE_ENABLE_FTS3=1 /D SQLITE_ENABLE_FTS4=1 /D SQLITE_ENABLE_FTS5=1 /D SQLITE_ENABLE_JSON1=1 /D SQLITE_ENABLE_COLUMN_METADATA=1 /D SQLITE_CORE=1 /I" + configure_module_dirname + "/../sqlite3/libsqlite /I" + configure_module_dirname);
-	
-	ADD_EXTENSION_DEP('pdo_sqlite', 'pdo');
-	// If pdo_sqlite is static, and sqlite3 is also static, then we don't add a second copy of the sqlite3 libs
-	if (PHP_PDO_SQLITE_SHARED || PHP_SQLITE3_SHARED || PHP_SQLITE3 == 'no') {
-		ADD_SOURCES(configure_module_dirname + "/../sqlite3/libsqlite", "sqlite3.c", "pdo_sqlite");
-	}
-}
+	if (CHECK_LIB("libsqlite3_a.lib;libsqlite3.lib", "pdo_sqlite", PHP_PDO_SQLITE) &&
+		CHECK_HEADER_ADD_INCLUDE("sqlite3.h", "CFLAGS_PDO_SQLITE") &&
+		CHECK_HEADER_ADD_INCLUDE("sqlite3ext.h", "CFLAGS_PDO_SQLITE")
```

Comment:
This could be generalized into a function like it is done for OpenSSL. Not critical for now.

Also shared might be a better choice to prefer for the shared extension build.

Reply:
Makes sense. I'll see to it.